src/api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios"
+
+const api = axios.create({ baseURL: "https://nc-news-de5p.onrender.com/api"})
+
+
+export const getArticles = () => {
+    return api.get("/articles").then((response) => {
+        return response.data
+    }).catch((err) => {
+        console.log(err)
+    })
+}
```

Comment:
Nice, but you should have a .catch at this level. If this does error then you are trying to deal with that error and resolve the promise which would go to the .then(...) of your home component which is no ideal. Therefore the .catch should be removed from here and added to the home component instead